django/contrib/auth/migrations/0011_update_proxy_permissions.py

```diff
@@ -39,7 +39,7 @@ def update_proxy_model_permissions(apps, schema_editor, reverse=False):
         old_content_type = proxy_content_type if reverse else concrete_content_type
         new_content_type = concrete_content_type if reverse else proxy_content_type
         try:
-            with transaction.atomic():
+            with transaction.atomic(using=schema_editor.connection.alias):
```

Comment:
We likely want to be forcing the usage of `schema_editor.connection.alias` in the `ContentType` query above and the `Permission` one below as well else they could be against different database.

We also want to skip this function entirely if `not router.allow_migrate('auth', 'permission')`

Reply:
Thanks, updated :+1:

Reply:
I added test for this case.